src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js

```diff
@@ -115,10 +115,14 @@ export default function ApplicationForm({
       })
       .then(res => {
         setProvinces(res.data.provinces.province);
+      })
+      .catch(error => {
+        setToError(true);
+        setError(error.response.status.toString());
```

Comment:
we need to figure out a way to display a more meaningful error to the user rather than just the response http status
need to discuss with jason / adrianna about this

Reply:
Yeah, I agree. Ideally we would display a specific error page view with a more easily understood description depending on the error.

Reply:
so will this be done at a later stage? not going to keep it as a blocker for this pr